src/composables/useWindowingConfigInitializer.ts

```diff
@@ -30,7 +59,8 @@ function useAutoRangeValues(imageID: MaybeRef<Maybe<string>>) {
 
     // Pre-compute the auto-range values
     const scalarData = imageData.value.getPointData().getScalars();
-    const [min, max] = scalarData.getRange();
+    // Assumes all data is one component
+    const { min, max } = fastComputeRange(scalarData.getData());
```

Comment:
Was the original getRange() call causing any particular issue?

Reply:
It was working perfectly for one component, which caused NANs in downstream windowing calculations.

Reply:
I'll try it

Reply:
`scalarData.getRange(0)` is the default and what was happaning before with just `scalarData.getRange()`.  That is a problem when we compute the `autoRangeValues`, which computes the histogram from all components from `scalarData.getData()`.  Histogram including all compnents does not make sense to me eather.  

We are rendering all components.  Looks like a single Window and Level are getting applied to all components:
https://github.com/Kitware/vtk-js/blob/master/Sources/Rendering/OpenGL/ImageMapper/index.js#L638-L655

I see this PR as a cludge to keep the app trucking along while still sort of supporting RGB images.

Reply:
In that case, this is fine. We will have to revisit this at some point.